### PR TITLE
the ref.go has been deleted

### PR DESCRIPTION
**Pull request by timyinshi**
the ref.go file has been deleted

/kind bug
/sig docs

### CODE REVIEW

**Comment by lavalamp** [comment]
Did the contents get moved, though?

File: keps/sig-api-machinery/20190711-remove-selflink.md [hunk 1]
```diff
@@ -134,8 +134,6 @@ I went through a k/k repo (including its staging repos) and all repos under [kub
 and this is the list of places that reference `SelfLink` fields (I excluded tests and all places
 in apiserver responsible for setting it):
 
-- [ ] https://github.com/kubernetes/kubernetes/blob/master/pkg/api/ref/ref.go
```
Comment (wojtek-t) [inline keps/sig-api-machinery/20190711-remove-selflink.md]: There were two copies of that file in the codebase - this one and staging/src/k8s.io/client-go/tools/reference/ref.go

I migrated stuff to the latter and removed this one.

Comment (wojtek-t) [reply keps/sig-api-machinery/20190711-remove-selflink.md]: I wouldn't removed it from the KEP though - it was there when the KEP was created.

Comment (timyinshi) [reply keps/sig-api-machinery/20190711-remove-selflink.md]: ok,this is fine.

**Pull request closed without merging**